package.json

```diff
@@ -37,6 +39,7 @@
     "url": "git://github.com/nodesecurity/nsp.git"
   },
   "scripts": {
+    "lint": "eslint . bin/eslint",
```

Comment:
Sorry, that should have been "bin/nsp" (since it doesn't have an extension and probably was unchecked by ESLint).